app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/AcceptOfferDialog.scala

```diff
@@ -53,8 +53,10 @@ class AcceptOfferDialog {
       val (oracleKey, eventId) = offer.contractInfo.oracleInfo match {
         case OracleInfoV0TLV(announcement) =>
           (announcement.publicKey.hex, announcement.eventTLV.eventId)
-        case _: MultiOracleInfoTLV =>
-          throw new RuntimeException("This is impossible.")
+        case multi: MultiOracleInfoTLV =>
```

Comment:
is there really much of an improvement to display 1 rather than all of them? I think this is misleading and will likely be harder for us to debug/trackdown later. I prefer the loud exception until we have end to end support